doc/changes/package_diffs/1.2.0/diff_standard-EXASOL-6.2.0_standard-EXASOL-6.2.0/README.md

```diff
@@ -0,0 +1,5 @@
+# Package Version Comparison between Standard-EXASOL-6.2.0 flavor in 1.2.0 and Standard-EXASOL-6.2.0 flavor in 1.1.0
+
```

Comment:
Did you create the comparison only for the changes package lists? I think, we should also add the comparison for the other once, also if they are equal.